src/main/app/pdf/mappers/theirDetailsMapper.ts

```diff
@@ -17,6 +17,7 @@ export class TheirDetailsMapper {
       address: {
         lineOne: party.address.line1,
         lineTwo: party.address.line2,
+        lineThree: party.address.line3,
```

Comment:
why we keep doing it :D ?

Reply:
doing what? I saw I had missed the line3 from a previous story so just added it in here :)

Reply:
No, I mean "lineThree" and "line3".

I though we had unify it.